core/geometry/Linac/BeamCollimation/MLC/MlcHD120.cc

```
@@ -0,0 +1,760 @@
+    G4ThreeVector central_leaf_offset_left(0., 3.5*cm, 2.52*mm);
+    G4ThreeVector central_leaf_offset_right(0.*cm, 3.5*cm, -2.87*mm);
+    
+    if (type.compare("SideA") == 0){
+        return createLeafShape(side_leaf_width, side_leaf_offset_left,side_leaf_offset_right);
+    } else if (type.compare("SideB") == 0){
+        return createLeafShape(side_leaf_width, side_leaf_offset_right, side_leaf_offset_left);
+    } else if (type.compare("CentralA") == 0){
+        return createLeafShape(central_leaf_width, central_leaf_offset_left,central_leaf_offset_right);
+    } else if (type.compare("CentralB") == 0){
+        return createLeafShape(central_leaf_width, central_leaf_offset_right, central_leaf_offset_left);
+    }
```

Cppcheck (2.19.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)